PeliculasRecomendadas.cpp

```diff
@@ -0,0 +1,124 @@
+//
+// Created by daseos on 28/6/20.
+//
+
+#include "PeliculasRecomendadas.h"
+
+PeliculasRecomendadas(){
+    listaPeliculasRecomendadas = 0;
+}
+
+bool PeliculasRecomendadas::actorEncontrado(Pelicula *peliculaNoVista, Lista<string> *actoresVistos){
+    Lista<string> *listaAuxiliar = peliculaNoVista->obtenerActores;
+    bool actorEnLista = false;
+    int i = 0;
+    while( (i < listaAuxiliar->obtenerCantidadElementos) && !actorEnLista ){
+        string actorLeido = listaAuxiliar->obtenerDato(i);
+        if(datoEnLista(actorLeido, actoresVistos)){
+            actorEnLista = true;
+        }
+        i++;
+    }
+    return actorEnLista;
+}
```

Comment:
Con el metodo datoEnLista() ubicado en la clase Lista, podriamos hacer algo asi
```suggestion
bool PeliculasRecomendadas::actorEncontrado(Pelicula *peliculaNoVista, Lista<string> *actoresVistos){
    Lista<string> *listaAuxiliar = peliculaNoVista->obtenerActores();
    bool actorEnLista = false;
    int i = 0;
    while( (i < listaAuxiliar->obtenerCantidadElementos) && !actorEnLista ){
        string actorLeido = listaAuxiliar->obtenerDato(i);
        actorEnLista = actoresVistos->datoEnLista(actorLeido);
        i++;
    }
    return actorEnLista;
}
```
Otra forma podría ser sacar el método, y en donde corresponda hacer
`listaPeliculas->obtenerDato(i)->obtenerActores()->datoEnLista(actor);`

O para que no quede tan largo algo como: 
```
Pelicula* pelicula = listaPeliculas->obtenerDato(i);
pelicula->obtenerActores()->datoEnLista(actor);
```